xml/System/Guid.xml

```diff
@@ -1270,7 +1272,7 @@ The <xref:System.Guid.ParseExact%2A> method requires the read-only character spa
 ## Examples  
  The following example calls the <xref:System.Guid.ToString%28System.String%29> method with each of the supported format specifiers to generate an array of strings that represent a single GUID. These are then passed to the <xref:System.Guid.ParseExact%2A> method, which successfully parses only the string that conforms to the "B" format specifier.  
   
- [!code-csharp[System.Guid.Parse#4](~/samples/snippets/csharp/VS_Snippets_CLR_System/system.guid.parse/cs/parseexactex1.cs#4)]
+ [!code-csharp-interactive[System.Guid.Parse#4](~/samples/snippets/csharp/VS_Snippets_CLR_System/system.guid.parse/cs/parseexactex1.cs#4)]
```

Comment:
this is missing the note. Add it to the MemberGroup element.

Reply:
@mairaw -- Maira, I'm not sure where to add the interactivity note for this example (Parse#4). I've put an INCLUDE (new line 1288) just before this example -- the nearest previous MemberGroup is new line 709, for "Equals".

Reply:
As per Parse#3, above.